frc_characterization/logger_analyzer/data_analyzer.py

```diff
@@ -962,26 +962,33 @@ def defineTestResults(*args):
         trackWidthEntry.configure(state="disabled")
         kGEntry.configure(state="disabled")
         kCosEntry.configure(state="disabled")
-        if STATE.test.get() != "Drivetrain":
+        if STATE.test.get() != Tests.DRIVETRAIN.value:
             dirMenu = OptionMenu(topFrame, STATE.subset, *sorted(directions))
         else:
             dirMenu = OptionMenu(topFrame, STATE.subset, *sorted(subsets))
         dirMenu.configure(width=20, state="normal")
         dirMenu.grid(row=0, column=7)
         STATE.subset.set("")
-        if STATE.test.get() == "Drivetrain" or STATE.test.get() == "Elevator":
+        if (
+            STATE.test.get() == Tests.DRIVETRAIN.value
+            or STATE.test.get() == Tests.ELEVATOR.value
+        ):
             diamEntry.configure(state="normal")
-            if STATE.test.get() == "Drivetrain":
+            if STATE.test.get() == Tests.DRIVETRAIN.value:
                 trackWidthEntry.configure(state="readonly")
             else:
                 kGEntry.configure(state="readonly")
         else:
             diamEntry.configure(state="disabled")
-            if STATE.test.get() == "Arm":
+            if STATE.test.get() == Tests.ARM.value:
                 kCosEntry.configure(state="readonly")
 
     def isRotation(units):
-        return units == "Rotations" or units == "Degrees" or units == "Radians"
+        return (
```

Comment:
```suggestion
		# note, this may be better to do Units(units) before calling isRotation
        return Units(units) in (Units.ROTATIONS, Units.DEGREES, Units.RADIANS)
```

Reply:
[Done](https://github.com/Piphi5/frc-characterization/blob/Refactor/frc_characterization/logger_analyzer/data_analyzer.py#L989)